index.bs

```diff
@@ -6604,10 +6607,10 @@ of user tracking:
     User agents may require the user to authorize access to databases
     before a site can use the feature.
 
-: Origin-tracking of stored data
+: Storage Key tracking of stored data
 ::
-    User agents may record the [=/origins=] of sites that contained content
-    from third-party origins that caused data to be stored.
+    User agents may record the [=/storage key=]s of sites that contained content
+    from third-party [=/storage key=]s that caused data to be stored.
```

Comment:
Does this section describing "third-party origins" still make sense when storage keys are involved?

I'd leave this section alone (i.e. revert these changes) and perhaps add a paragraph about storage keys partitioning third-party storage? Or leave a TODO somewhere (new issue?) to add it.